app/src/main/java/com/cornellappdev/uplift/models/UpliftActivity.kt

```diff
@@ -0,0 +1,65 @@
+package com.cornellappdev.uplift.models
+
+data class UpliftActivity(
```

Comment:
Write a spec for this guy! Even if the spec is something quick & simple, this'll be a pretty important class in the future, so it's best to make sure it's all tidied up.